PocketKit/Sources/Analytics/AppEvents/Tags.swift

```diff
@@ -136,4 +136,26 @@ public extension Events.Tags {
             )
         )
     }
+
+    /// Fired when a user adds tag to an input list in the `Add Tags` screen for an item
+    static func renameTag() -> Event {
```

Comment:
Im curious for analytics purposes do they need to know the stuff being renamed?

Reply:
let me check with them, I was under the assumption that it could be useful to keep track of how many users are using these functionality.

Reply:
ooooh sorry I misread that, I can follow up with her on that. Not sure what purposes that would serve.

Reply:
yea no biggie. I was just curious, prob doesn't matter im sure they can corelate on the data model if ever needed.

Reply:
okay, let me bring a bigger question to her since don't think we're doing that for other tag tracking calls either. will do a follow up PR if anything